Dockerfile.s390x

```diff
@@ -1,15 +0,0 @@
-FROM s390x/alpine:3.10.1
-
-ARG COMPOSE_VERSION=1.16.1
-
-RUN apk add --update --no-cache \
-    python \
-    py-pip \
-  && pip install --no-cache-dir docker-compose==$COMPOSE_VERSION \
-  && rm -rf /var/cache/apk/*
-
-WORKDIR /data
-VOLUME /data
-
-
-ENTRYPOINT ["docker-compose"]
```

Comment:
This file is not used by the project...

Reply:
It would be better to adapt the other Dockerfile if we want s390x support than use this so I'm happy to remove